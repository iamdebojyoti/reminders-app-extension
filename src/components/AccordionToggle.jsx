import { Button } from "react-bootstrap";
import { useAccordionButton } from 'react-bootstrap/AccordionButton';
import { CaretDown } from "./Icons";

const AccordionToggle = ({eventKey, variant}) => {
  const toggle = useAccordionButton(eventKey);

  return (
    <Button id={"acrdn-" + eventKey} className="float-end" variant={variant} onClick={toggle}>
        <CaretDown/>
    </Button>
  );
}

export default AccordionToggle;