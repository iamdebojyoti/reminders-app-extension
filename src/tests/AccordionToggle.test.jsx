import { shallow } from "enzyme";
import AccordionToggle from "../components/AccordionToggle";

describe('Testing <AccordionToggle/> component', () => {

    let component;

    it('should match snapshot of AccordionToggle', () => {
      component = shallow(<AccordionToggle eventKey="1" variant="success"/>);
      expect(component.debug()).toMatchSnapshot();
    });
});