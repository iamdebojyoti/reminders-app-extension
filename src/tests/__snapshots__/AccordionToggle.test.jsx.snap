// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <AccordionToggle/> component should match snapshot of AccordionToggle 1`] = `
"<Button id=\\"acrdn-1\\" className=\\"float-end\\" variant=\\"success\\" onClick={[Function (anonymous)]} active={false} disabled={false}>
  <CaretDown />
</Button>"
`;
